packages/react-native-editor/bin/extract-used-strings.js

```diff
@@ -164,7 +164,10 @@ if ( require.main === module ) {
 		domains.push( pluginName );
 	}
 
-	const potFilesDir = path.resolve( path.dirname( outputFile ) );
+	const potFilesDir = path.join(
+		path.resolve( path.dirname( outputFile ) ),
+		'pot-files'
+	);
```

Comment:
The POT files generated will be under a new subfolder, this way we can easily remove them after the script finishes.